app/controllers/extra_registrations_controller.rb

```diff
@@ -5,7 +5,7 @@ class ExtraRegistrationsController < ApplicationController
   STEP_FORMS = {
     # partial => form object class
     name: Users::NameForm,
-    setting: Users::SettingForm
+    setting: Users::SettingForm,
```

Comment:
Who cares about the last element in a hash having a comma.

Reply:
I have experienced the pedantry paying off when I shuffle lines and move the last one up, but forget to add on a comma. However that is highlighted by everything breaking as a result.